src/routes/plans/[id]/+page.svelte

```diff
@@ -217,10 +257,12 @@
       >
         <GearWideConnectedIcon />
       </NavButton>
-      <NavButton title="View">
-        <ColumnsIcon />
-        <ViewMenu slot="menu" />
-      </NavButton>
+      <ViewMenu
+        on:create-view={onCreateView}
```

Comment:
Nit pick. Most (if not all) custom dispatch events use camelCase so we should probably just prefer camel case here. Not sure if there is an easy way to lint for this.